### ISSUE TITLE
Hello unam, help please

### ISSUE THREAD

**Issue by ssska12** [UnamSanctam/SilentCryptoMiner#672]
RVN mining is not working what to do? @UnamSanctam

**Comment by UnamSanctam** [comment]
In what way is it not working? Can you run the checker and post what it returns?

**Comment by ssska12** [comment]
![image](https://user-images.githubusercontent.com/104590176/165818882-301b940b-750f-4c2d-98d2-baf587a2212b.png)

**Comment by UnamSanctam** [comment]
And you don't seen any GPU usage in something like MSI Afterburner?

**Comment by ssska12** [comment]
When I test everything works for me, but when I distribute it to others no

**Comment by UnamSanctam** [comment]
Do they have a GPU that can mine it? And are you sure they install it (that their antivirus doesn't remove it)?

**Comment by ssska12** [comment]
I do not think that out of 20 people with at least 4 GB of video cards will not work for at least 1

**Comment by ssska12** [comment]
my build detect 1/26

**Comment by UnamSanctam** [comment]
If it's working for you then it will also work for others (your settings also look correct), so either of those would be the reason (not enough VRAM or being removed). You do also have miners to your address mining, keep in mind that all miners will use the worker name you set (rig).

**Comment by ssska12** [comment]
I don't know why, but the hash is not growing

**Comment by UnamSanctam** [comment]
You can add a CPU miner to know if they're actually installing your miner, since the CPU miner will run on almost any computer. Also, you can use {COMPUTERNAME} as the worker name which will cause the miner to use the computers name as the worker name.

**Comment by ssska12** [comment]
wallet.{COMPUTERNAME} ?

**Comment by ssska12** [comment]
But still could you check rvn mining?

**Comment by UnamSanctam** [comment]
It's working correctly, as you can also see since it mines correctly on your computer.

**Comment by ssska12** [comment]
well, everything is fine with me, but when you distribute the hash does not grow

**Comment by UnamSanctam** [comment]
Yes most likely due to either of the two reasons I listed earlier, either they probably don't have a good enough GPU or their antivirus could be removing it. There's no real difference from running it on your own computer compared to running it on another one.

**Comment by ssska12** [comment]
well, this is some kind of nonsense, everything is fine with etc, but not with rvn

**Comment by UnamSanctam** [comment]
How would you know? But RVN (the KawPow algorithm) is much more intensive than ETC so it can cause lag for the users as well. Also it uses 3.35GB of VRAM so many 4GB cards cannot mine it, since Windows and other programs can use up to 1GB of VRAM on most computers.
